parcels/field.py

```diff
@@ -375,9 +375,8 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
                                                         dimensions, indices, netcdf_engine)
             kwargs['dataFiles'] = dataFiles
 
-        field_chunksize = kwargs.get('field_chunksize', None)
-        if field_chunksize and grid.master_chunksize is None:
-            grid.master_chunksize = field_chunksize
+        chunksize = kwargs.get('chunksize', None)
+        grid.chunksize = chunksize
```

Comment:
yes, nice - this way, with Field.chunksize and Grid.chunksize are clear -> less confusing

Reply:
Well, Ideally I would have wanted to get rid of `grid.chunksize` and only have `field.chunksize`, but then we can't check for duplicate grids in https://github.com/OceanParcels/parcels/blob/fixing_dask_namemaps/parcels/gridset.py#L33-L37. If you have a way to check the chunking there without need for the `grid.chunksize`, then we can get rid of the line `grid.chunksize = chunksize`

Reply:
no no - misunderstanding. I just said that this change the way it is now, that is fine.

Reply:
Well, but I would still prefer to get rid of `grid.chunksize`, just can't figure out how. My question to you @CKehl was: can you think of a way to get rid of `grid.chunksize` and still do the check in https://github.com/OceanParcels/parcels/blob/fixing_dask_namemaps/parcels/gridset.py#L33-L37?

Reply:
well, I needed to add that to prevent adding fields (via `add_fields(...)`) to the grid even though the grid was initialised with a different chunksize already. I'll look into that.

Reply:
so, the short answer is: because there is no way to get the field(s) of a grid fromout the `Grid` or `GridSet` itself, we cannot get rid of `Grid.chunksize` without also loosing the failsafe-check.